getting-started/cookbooks/webinar1-myapp/templates/default/index.php.erb

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <?php
```

Comment:
The <?php ?> block has to be in line 1, before DOCTYPE

Reply:
fixed